go.mod

```diff
@@ -1,69 +1,73 @@
 module github.com/ori-edge/k8s_gateway
 
-go 1.18
+go 1.21
 
 require (
 	github.com/coredns/caddy v1.1.1
-	github.com/coredns/coredns v1.10.1
-	github.com/miekg/dns v1.1.54
+	github.com/coredns/coredns v1.11.1
+	github.com/miekg/dns v1.1.55
 	github.com/nginxinc/kubernetes-ingress v1.12.5
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/client-go v0.26.3
-	sigs.k8s.io/gateway-api v0.4.3
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/client-go v0.28.1
+	sigs.k8s.io/gateway-api v0.8.0
```

Comment:
should probably bump this since it has recently been updated

Reply:
@larivierec Thanks, updated.